src/drone_base.cpp

```diff
@@ -486,17 +326,17 @@ void DroneBase::joy_callback(const sensor_msgs::msg::Joy::SharedPtr msg)
   prev_msg = *msg;
 }
 
-void DroneBase::tello_response_callback(const tello_msgs::msg::TelloResponse::SharedPtr msg)
+void DroneBase::tello_response_callback(tello_msgs::msg::TelloResponse::SharedPtr msg)
 {
-  ActionMgr::State result = action_mgr_->complete(msg);
+  ActionMgr::State result = action_mgr_->complete(std::move(msg));
```

Comment:
Ah, I was wondering why you removed "const" from the message parameter. Very nice.